jamaljsr/polar
# fix(lightningd): display CA cert and distinguish it from client cert and key

### Description

This PR moves `TLS Cert` and `TLS Key` to `TLS Client Cert` and `TLS Client Key` in order to avoid confusion with the server certificate+key or the CA certificate+key. It changes `TLS Cert` to the `ca.pem` certificate instead of the `client.pem` certificate.

Translations are also included for the new strings.

### Steps to Test

1. Create a new network
2. Add a Core Lightning Node
3. Go to the Connect tab
4. You can check the displayed certificates are identical with the files certificates:
```shell
$ xxd -p <path-to-polar-network>/volumes/c-lightning/<node name>/lightningd/regtest/ca.pem | tr -d '\n'
$ xxd -p <path-to-polar-network>/volumes/c-lightning/<node name>/lightningd/regtest/client.pem | tr -d '\n'
$ xxd -p <path-to-polar-network>/volumes/c-lightning/<node name>/lightningd/regtest/client-key.pem | tr -d '\n'
```

### Screenshots

<img width="250" alt="image" src="https://user-images.githubusercontent.com/38882571/184695729-9e035e20-d79c-41cc-8226-757eae27511d.png"><img width="250" alt="image" src="https://user-images.githubusercontent.com/38882571/184695767-4108b36e-dd5d-4f3c-8a7b-92d329edd7f1.png"><img width="250" alt="image" src="https://user-images.githubusercontent.com/38882571/184695801-9ea88fc1-fb54-43a7-9581-d5ea7800fc23.png">
